### PR TITLE
Milestone-1

### PR DESCRIPTION
**Pull request by Paul-Clue**
## What I Did:
- I implemented all initial configurations for the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
```
Comment (bellom) [inline README.md:3]: **Let's make your repo look more professional by updating your `README` file**.
Checkout out the below link for help, feel free to follow this [Microverse template](https://github.com/microverseinc/readme-template) from pathwright

Make sure to include Setup and Install section.

File: package.json [hunk 2]
```diff
@@ -34,5 +34,21 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
```
Comment (bellom) [inline package.json:39]: Kindly and `big.js` and `prop-types` to `dependencies `
image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Paul-Clue  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check my comments under this review;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Review by voscarmv — APPROVED** [review]
# Project status: Approved 💯 ✨ 😎 👍 
### You may merge this branch and move forward ⛙🛣️ 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**